modules/apps/object/object-rest-api/src/main/java/com/liferay/object/rest/openapi/v1_0/ObjectEntryOpenAPIResource.java

```diff
@@ -36,6 +37,7 @@ public Response getOpenAPI(
 			HttpServletRequest httpServletRequest, String type, UriInfo uriInfo)
 		throws Exception;
 
-	public Map<String, Schema> getSchemas() throws Exception;
+	public Map<String, Schema> getSchemas(ObjectDefinition objectDefinition)
```

Comment:
We talked about this internally. This is unnecessary because it is attaching an external object definition to an ObjectEntryOpenAPIResource of another object definition.

I mean, if we want to retrieve the schemas of the Student object, we use the objectEntryOpenAPIResource of the Student. If we want the schemas of the University object, we use the objectEntryOpenAPIResource of the University.
It doesn't make sense to use the Student's objectEntryOpenAPIResource to get the schemas of the University.

We can get the proper ObjectEntryOpenAPIResource with the ObjectEntryOpenAPIResourceProvider, which ask for an objectDefinition, exactly what you are providing here.